zssh/zssh/main.go

```diff
@@ -33,13 +33,17 @@ import (
 const ExpectedServiceAndExeName = "zssh"
 
 var (
-	flags = zsshlib.SshFlags{}
+	flags   = zsshlib.SshFlags{}
+	version = "v0.0.0"
+	commit	= "unknown"
+	date	= "unknown"
 )
 
 var rootCmd = &cobra.Command{
 	Use:   fmt.Sprintf("%s %s <remoteUsername>@<targetIdentity>", ExpectedServiceAndExeName, flags.ServiceName),
 	Short: "Z(iti)ssh, Carb-loaded ssh performs faster and stronger than ssh",
 	Long:  "Z(iti)ssh is a version of ssh that utilizes a ziti network to provide a faster and more secure remote connection. A ziti connection must be established before use",
+	Version: fmt.Sprintf("%s (commit: %s, built: %s)", version, commit, date),
```

Comment:
can you update zscp, too, please? i'm not a fan of all the additional output for the commit hash and date. i'd rather just see the version number. 

Can you make the output look like:
```
1.0.9B (built:2024-09-13T19:26:54Z)
```
or with the hash use the short hash:
```
1.0.9B (built:2024-09-13T19:26:54Z hash:e9df4770)
```

Reply:
i see you did zscp too - thx

Reply:
![image](https://github.com/user-attachments/assets/759bd9a8-1f08-490b-8988-963296a6edae)